website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -33,9 +33,14 @@ A notification configuration specifies a destination URL, a payload type, and th
 
    - **Email Recipients** This is only available for emails. Select users that should receive notifications.
 
-   - **Workspace Events:** Terraform Cloud can send notifications for all events or only for specific events. The following events are available:
-       - **Drift**: Terraform Cloud detected configuration drift. This notification is only available if [drift detection](/cloud-docs/workspaces/settings/drift-detection) is enabled for the workspace.
-       - **Drift Check Fail**: A drift detection assessment failed. This notification is only available if [drift detection](/cloud-docs/workspaces/settings/drift-detection) is enabled for the workspace.
+       <!-- BEGIN: TFC:only name:drift-detection -->
+
+    -   **Workspace Events:** Terraform Cloud can send notifications for all events or only for specific events. The following events are available:
+        -   **Drift**: Terraform Cloud detected configuration drift. This notification is only available if [drift detection](/enterprise/workspaces/settings/drift-detection) is enabled for the workspace.
+        -   **Drift Check Fail**: A drift detection assessment failed. This notification is only available if [drift detection](/enterprise/workspaces/settings/drift-detection) is enabled for the workspace.
+
+     <!-- END:   TFC:only name:drift-detection -->
+    
```

Comment:
@laurapacilio There's some inconsistent indentation/leading whitespace with these two content-exclusion comments that looks to be causing the build to fail. 

For me, this also causes the local dev preview to return an error page

Reply:
fixed - thank you!!!